src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int hypotenuse;

```

Comment:
The hypotenuse field is not needed in the RightTriangle class because it can be calculated from the legs of the right triangle using the Pythagorean theorem. Consider removing it to simplify the class design.